src/ui_ng/src/app/config/config.component.ts

```diff
@@ -266,7 +266,7 @@ export class ConfigurationComponent implements OnInit, OnDestroy {
                         this.msgHandler.handleReadOnly();
                     }
 
-                    if (changes['read_only'].toString() === "false") {
+                    if (changes['read_only'] && changes['read_only'].toString() === "false") {
```

Comment:
Incorrect statement logic

Reply:
'changes' only contains number and string. so there are no boolean problems.